src/components/elements/tableFilters/filters/FilterMenu.tsx

```diff
@@ -15,34 +15,38 @@ export interface TableFilterMenuProps<T> {
   setFilterValues: (value: Partial<T>) => any;
 }
 
-const TableFilterMenu = <T,>(props: TableFilterMenuProps<T>): JSX.Element => {
+const TableFilterMenu = <T,>({
+  filters,
+  filterValues,
+  setFilterValues,
+}: TableFilterMenuProps<T>): JSX.Element => {
```

Comment:
The diff in this file is no longer necessary since I moved the transformation to the separate util. But I think it makes it more readable to destructure, so keeping the change